de-lib/src/main/java/org/iplantc/de/apps/client/presenter/categories/AppCategoriesPresenterImpl.java

```diff
@@ -189,6 +196,30 @@ public void onSuccess(List<AppCategory> result) {
 
     }
 
+    @Override
+    public void checkForAgaveRedirect() {
+        if (!userInfo.hasSessionError()) {
+            return;
+        }
+
+        oauthServiceFacade.getRedirectUris(new AsyncCallback<Map<String, String>>() {
```

Comment:
It looks like this `checkForAgaveRedirect` will only display the `AgaveAuthPrompt` if the `bootstrap` endpoint had an error checking for this redirect URI at login.

The `DEAppsWindow#afterShow` method used to show this prompt if `userInfo` had an Agave redirect URI at login from the `bootstrap` response (now this method just calls `checkForAgaveRedirect`, and it doesn't look like that's called from anywhere else).

If `userInfo.hasSessionError()` is `false` above, then we should probably still display the prompt if `userInfo` has an Agave redirect URI from `bootstrap`.